lms/djangoapps/discussion/templates/discussion/discussion_board_fragment.html

```diff
@@ -131,7 +131,7 @@ <h3>${course_organization}</h3>
                 </main>
                 <aside class="forum-nav" role="complementary" aria-label="${_("Discussion thread list")}">
                     <%include file="_filter_dropdown.html" />
-                    <div class="discussion-thread-list-container"></div>
+{#                    <div class="discussion-thread-list-container"></div>#}
```

Comment:
What is this for?

Reply:
This was sidebar for default discussion which was hidden on page load, I just commented so this does not render and saves time to load the page.

Reply:
Just make sure it works with @mumarkhan999 changes as well.